hrms/hr/doctype/leave_application/leave_application.py

```diff
@@ -588,7 +588,7 @@ def notify_employee(self):
 			frappe.msgprint(_("Please set default template for Leave Status Notification in HR Settings."))
 			return
 		email_template = frappe.get_doc("Email Template", template)
-		message = frappe.render_template(email_template.response, args)
+		message = frappe.render_template(email_template.response_, args)
```

Comment:
Email template has a property `response_` now that sets the exact response field. Weird naming though 🫠

<img width="943" alt="image" src="https://github.com/frappe/hrms/assets/24353136/1535c2e9-a777-491a-97d8-b6a435820d6f">